common/djangoapps/student/management/commands/tests/test_force_send_notification_digest.py

```diff
@@ -0,0 +1,17 @@
+"""
+Tests for the Django management command force_send_digest
+"""
+
+import mock
+from django.conf import settings
+from django.test import TestCase
+from student.management.commands import force_send_notification_digest
+
+
+@mock.patch.dict(settings.FEATURES, {'ENABLE_NOTIFICATIONS': True})
+class ForceSendDigestCommandTest(TestCase):
+    def test_command_check(self):
+        # run the management command for purging notifications.
+        force_send_notification_digest.Command().handle(**{'send_daily_digest': True, 'send_weekly_digest': True, 'namespace': 'All'})
+        force_send_notification_digest.Command().handle(**{'send_daily_digest': True, 'send_weekly_digest': True, 'namespace': 'ABC'})
+        # force_send_digest.Command().handle(**{'send_daily_digest': False, 'send_weekly_digest': False, 'namespace': 'ABC'})
```

Comment:
This line is commented out -- if it doesn't need to be there we should just remove (that's what source control is for).  If it does need to be there, but should be commented except for a particular scenario, then a descriptive comment should probably be added.